### PR TITLE
feat: Solana Coming soon

### PR DESCRIPTION
**Pull request by namgold**
Signed-off-by: Nam Nguyen <nam.nguyen@kyber.network>

### CODE REVIEW

**Review by nguyenhoaidanh — APPROVED** [review]

File: src/components/Menu/index.tsx [hunk 1]
```diff
@@ -144,8 +144,6 @@ const ClaimRewardButton = styled(ButtonPrimary)`
 export const NewLabel = styled.span`
   font-size: 10px;
   color: ${({ theme }) => theme.red};
-  height: calc(100% + 4px);
-  margin-left: 2px;
```
Comment (namgold) [inline src/components/Menu/index.tsx:148]: useless, so removed

File: src/components/NetworkModal/index.tsx [hunk 2]
```diff
@@ -58,27 +60,41 @@ export const ListItem = styled.div<{ selected?: boolean }>`
...
+  `}
+`
+
+const NewLabel = styled.div`
+  position: absolute;
+  top: 7px;
+  left: 93px;
+  font-size: 8px;
+  font-weight: 500;
+  color: #ff537b;
```
Comment (viet-nv) [inline src/components/NetworkModal/index.tsx:95]: hard code?

Comment (namgold) [reply src/components/NetworkModal/index.tsx:95]: What's issue

Comment (namgold) [reply src/components/NetworkModal/index.tsx:95]: This one not export anywhere, only use here

Comment (namgold) [reply src/components/NetworkModal/index.tsx:95]: I cloned it from [here](https://github.com/KyberNetwork/kyberswap-interface/blob/main/src/components/YieldPools/styleds.tsx#L143-L150) with some customized

File: src/components/Menu/index.tsx (hunk 1 above)
Comment (viet-nv) [reply src/components/Menu/index.tsx:148]: is this affected?
image

**Review by viet-nv — APPROVED** [review]

**Pull request merged**